contracts/src/v0.8/KeeperRegistry.sol

```diff
@@ -834,7 +862,15 @@ contract KeeperRegistry is
     success = _callWithExactGas(params.gasLimit, upkeep.target, callData);
     gasUsed = gasUsed - gasleft();
 
-    uint96 payment = _calculatePaymentAmount(gasUsed, params.adjustedGasWei, params.linkEth);
+    uint256 l1CostWei = 0;
```

Comment:
this function is called with actual performData so use msg.data and getCurrentTxL1GasFees